homeassistant/components/devolo_home_control/sensor.py

```diff
@@ -167,6 +167,12 @@ def __init__(
         self._sensor_type = consumption
         self._device_class = DEVICE_CLASS_MAPPING.get(consumption)
 
+        if consumption == "total":
+            self._attr_state_class = "measurement"
+            self._attr_last_reset = device_instance.consumption_property[
+                element_uid
+            ].total_since
```

Comment:
Just for validation, total_since is a datetime object, right?

Reply:
Yes it is.